test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -65,7 +65,8 @@
 
 #define MAX_PKTS                 1000u
 #define PKT_BUF_SIZE             1460
-#define MAX_PAYLOAD              1400
+#define MIN_HDR_LEN              (ODPH_ETHHDR_LEN + ODPH_UDPHDR_LEN + ODPH_IPV4HDR_LEN)
+#define MAX_PAYLOAD              (PKT_BUF_SIZE - MIN_HDR_LEN)
```

Comment:
Since the code is a bit messy with respect to the packet length and contains hidden assumptions, it might be good to check payload_len against MAX_PAYLOAD in make_pkt().